README.md

```diff
@@ -192,6 +192,16 @@ datasources:
     basicAuthPassword: test_password
 ```
 
+## Getting Help
+
+If you have any questions or feedback regarding Loki:
+
+- Ask a question on the Loki Slack channel. To invite yourself to the Grafana Slack, visit [http://slack.raintank.io/](http://slack.raintank.io/) and join the #loki channel.
+- [File an issue](https://github.com/grafana/loki/issues/new) for bugs, issues and feature suggestions.
```

Comment:
I'd add:

Note that UI issues should be filed directly in [Grafana](https://github.com/grafana/grafana/issues/new)